reana_job_controller/kubernetes_job_manager.py

```diff
@@ -316,6 +322,53 @@ def _add_krb5_init_container(self, secrets_volume_mount):
         self.job['spec']['template']['spec']['initContainers'].append(
             krb5_container)
 
+    def _add_proxy_init_container(self, secrets_volume_mount):
+        """Add  sidecar container for a job."""
+        ticket_cache_volume = {
+            'name': 'proxy-cache',
+            'emptyDir': {}
+        }
+        volume_mounts = [
+            {
+                'name': ticket_cache_volume['name'],
+                'mountPath': current_app.config['PROXY_CERT_CACHE_LOCATION']
+            }
+        ]
+
+        proxy_pass = os.environ.get('VOMSPROXY_PASS')
+        proxy_file_path = os.path.join(
+                               current_app.config['PROXY_CERT_CACHE_LOCATION'],
+                               current_app.config['PROXY_CERT_CACHE_FILENAME']
+                           )
+
+        proxy_container = {
+            'image': current_app.config['PROXY_CONTAINER_IMAGE'],
+            'command': ['/bin/bash'],
+            'args': ['-c', 'cp /etc/reana/secrets/userkey.pem /tmp/userkey.pem; \
+                     chmod 400 /tmp/userkey.pem; \
+                     echo {0} | base64 -d | voms-proxy-init \
+                     --voms cms --key /tmp/userkey.pem \
+                     --cert $(readlink -f /etc/reana/secrets/usercert.pem) \
+                     --pwstdin --out {1}; \
+                     chown {2} {1}'.format(proxy_pass, proxy_file_path, self.kubernetes_uid)],
```

Comment:
What do you think of using keywords (i.e. `{voms_proxy_pass}`) instead of the numbers (i.e. `{0}`) to reference the parameters? I think it could read a bit better since it is a long command :)